src/main/java/com/faforever/api/email/EmailService.java

```diff
@@ -50,4 +51,16 @@ public void sendActivationMail(String username, String email, String activationU
         String.format(registration.getHtmlFormat(), username, activationUrl)
     );
   }
+
+  @SneakyThrows
+  public void sendPasswordResetMail(String username, String email, String passwordResetUrl) {
+    PasswordReset passwordReset = properties.getPasswordReset();
+    emailSender.sendMail(
```

Comment:
PS.: I discovered latly the `@Builder` lombok annotation: https://projectlombok.org/features/Builder.html